Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -340,3 +340,37 @@
   - type: HeldSpeedModifier
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitScaf
+
+# Tactical Maid Hardsuit
+
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitTacticalMaid
+  name: tactical maid hardsuit
+  description: Multiple coats of stain and dust resistant compounds all built into a single suit.
+  components:
+  - type: Sprite
+    sprite: _NF/Clothing/OuterClothing/Hardsuits/tactical_hardsuit.rsi
+  - type: Clothing
+    sprite: _NF/Clothing/OuterClothing/Hardsuits/tactical_hardsuit.rsi
+  - type: Armor # Remains perfectly clean of stains
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Caustic: 0.05
+  - type: ExplosionResistance
+    damageCoefficient: 0.8
+  - type: PressureProtection
+    highPressureMultiplier: 0.5
+    lowPressureMultiplier: 1000
+  - type: ClothingSpeedModifier # So many layers of stain protection makes it very heavy
+    walkModifier: 0.55
+    sprintModifier: 0.55
+  - type: HeldSpeedModifier
+  - type: FootstepModifier
+    footstepSoundCollection:
+      path: /Audio/Mecha/sound_mecha_powerloader_step.ogg
```

Comment:
Does this work for you?  Doesn't seem to work for me.

Reply:
It needs to load into the feet slot to actually be applied.  SharedMoverController:452-457:
```csharp
            if (_inventory.TryGetSlotEntity(uid, "shoes", out var shoes) &&
                TryComp<FootstepModifierComponent>(shoes, out var modifier))
            {
                sound = modifier.FootstepSoundCollection;
                return true;
            }
```

Reply:
Honestly, wouldn't be opposed to extending that to outer clothing, especially since some of the suits (e.g. spationaut hardsuit, this) come with a set of boots in the sprite?

Reply:
oh i see, i think it will just remove it from now dont want to mess any code right now.